wire-ios-system/Source/DateProviding/CurrentDateProviding.swift

```diff
@@ -18,7 +18,8 @@
 
 import Foundation
 
-/// Abstracts accessing specific date/time values like `.now` in order to be mockable in unit tests.
-public protocol DateProviding {
+// sourcery: AutoMockable
+/// Abstracts accessing the current system date in order to be mockable in unit tests.
+public protocol CurrentDateProviding {
```

Comment:
The renaming could have been a single PR, but it's okay for me.

Reply:
I agree.